ksql-engine/src/test/java/io/confluent/ksql/TopologyFileGenerator.java

```diff
@@ -49,15 +52,25 @@
  */
 public final class TopologyFileGenerator {
 
-    private static final String BASE_DIRECTORY = "ksql-engine/src/test/resources/expected_topology/";
+    private static final String BASE_DIRECTORY = "src/test/resources/expected_topology/";
 
-    private TopologyFileGenerator() {
+    @Ignore // comment me out to generate the persisted topologies
+    @Test
+    public void doGenerateTopologies() throws Exception {
+        generateTopologies(BASE_DIRECTORY);
     }
 
-    public static void main(final String[] args) throws IOException, ParserConfigurationException, SAXException {
+    @Test
+    public void shouldGenerateTopologies() throws Exception {
```

Comment:
This test will take a while to run. So probably best to have this test marked as an `@IntegrationTest`.